src/resources/questions/multi.ts

```diff
@@ -207,6 +207,7 @@ function buildDropdownPart(part: any, _i: number) {
     ),
     objectives: skillrefs.map((s: any) => s.idref),
     scoringStrategy: 'average',
+    explanation: Common.maybeBuildPartExplanation(responses),
```

Comment:
I think we need to add this "multiple feedbacks to explanation" conversion to question types beyond just multipart.  Any question type could feature this.

Reply:
Good catch, updated this to include all question types